lib/shopify_api/resources/vault_session.rb

```diff
@@ -0,0 +1,28 @@
+module ShopifyAPI
+  module VaultSessionFormat
```

Comment:
> Because updating cardsink to include a root based on some criteria and/or update all the existing apps to use the new format is an enormous task.

Thanks for the explanation 👍 

> I can extract this into a separate file

Please, I feel this doesn't belong in the resources folder, but with the other helper modules. If somebody ever needs it for something else, they wouldn't know to look in `vault_session.rb`. Which means it would be duplicated.